cfme/scripting/conf.py

```diff
@@ -101,5 +103,24 @@ def decrypt(conf_name, delete, skip):
     print('{} conf decrypted'.format(conf_name))
 
 
+@main.command("write", help="Write a username and password into vault")
+@click.argument("env")
+@click.argument("username")
+@click.argument("password")
+def write(env, username, password):
```

Comment:
You can use `dynaconf` direclty to write:

```py
$ dynaconf write vault -s SECRET=this_goes_to_default_key -s FOO=another_default_key_value
$ dynaconf write vault -e development -s SECRET=this_is_stored_in_development_key
$ dynaconf write vault -e production -s SECRET=this_is_stored_in_production_key
```

Then you can now use `settings.from_env("production").SECRET` to read the proper value :)

If you really need to have it under `miq` custom name, as you are using click you can directly import from `dynaconf.cli`

Just import
https://github.com/rochacbruno/dynaconf/blob/7287fae4d133f82950829fb605dfe8d57fd30ef7/dynaconf/cli.py#L503

```py
from dynaconf.cli import write
# and add to your own click
```

Reply:
@rochabruno that's some handy info. Thanks :)